### PR TITLE
Bump to 7.7.2

### PR DESCRIPTION
**Pull request by thenexxuz**

### CODE REVIEW

File: gitkraken-aur/PKGBUILD [hunk 1]
```diff
@@ -40,6 +40,7 @@ package() {
     find "$pkgdir"/opt/gitkraken/ -type f -exec chmod 644 {} \;
     chmod 755 "$pkgdir"/opt/gitkraken/gitkraken
     chmod 755 "$pkgdir"/opt/gitkraken/resources/app.asar.unpacked/src/js/redux/domain/AskPass/AskPass.sh
+    chmod -R 755 "$pkgdir"/opt/gitkraken/resources/app.asar.unpacked/resources/cli/unix/
```
Comment (Azd325) [inline gitkraken-aur/PKGBUILD:43]: How you find it, or what fixes this?

Comment (thenexxuz) [reply gitkraken-aur/PKGBUILD:43]: It is something that I am not allowed to talk about yet ;-)

Comment (thenexxuz) [reply gitkraken-aur/PKGBUILD:43]: What I can say is that this will be needed to fix an issue with a new feature being worked on.

Comment (chuckdries) [reply gitkraken-aur/PKGBUILD:43]: Hi! I'm a developer on the GitKraken team. This fixes an issue for a semi-secret feature that's currently available in beta for a select group of users.

**Comment by Azd325** [comment]
Merged in 3ad8a30a455040fc996675c6fa299f811066042f

File: gitkraken-aur/PKGBUILD (hunk 1 above)
Comment (Azd325) [reply gitkraken-aur/PKGBUILD:43]: Ok. Then be please more specific what should get the executable flag and my first look not all files are execuables.

Comment (Azd325) [reply gitkraken-aur/PKGBUILD:43]: This part i excluded for the moment from the bumping to 7.2.2 but happy to merge later on.

**Pull request closed without merging**